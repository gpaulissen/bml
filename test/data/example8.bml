// https://github.com/gpaulissen/bml/issues/4
// strange characters for laTeX: & % $ # _ { } ~ ^ \

1N-2C PUP to 2!d
  2D Forced
    P Weak with !d
    2H TRF
      2S
        2N 5+c !s, G/T
        3CD !s & minor, G/T (%$#_{}~^\)
