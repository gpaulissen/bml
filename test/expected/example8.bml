#VUL 00

#SEAT 0

1N
  2C = PUP to 2!d
    2D = *Forced*
      P = Weak with !d
      2H = TRF
        2S
          2N = 5+c !s, G/T
          3C = !s & minor, G/T (%$#_{}~^\)
          3D = !s & minor, G/T (%$#_{}~^\)
1C
  (1D)
    2C = support
    2D = cuebid
    2H = weak jump
    2S = weak jump
  (1H)
    2C = support
    2D = forcing
    2H = cuebid
    2S = weak jump
  (1S)
    2C = support
    2D = forcing
    2H = forcing
    2S = cuebid
