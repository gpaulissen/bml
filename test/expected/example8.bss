*00{}=NYYYYYY
001N=NYYYYYY0
001NP2C=NYYYYYY008PUP to 2!d
001NP2CP2D=NYYYYYY008Forced
001NP2CP2DPP=NYYYYYY0Weak with !d
001NP2CP2DP2H=NYYYYYY008TRF
001NP2CP2DP2HP2S=NYYYYYY008
001NP2CP2DP2HP2SP2N=NYYYYYY05+c !s, G/T
001NP2CP2DP2HP2SP3C=NYYYYYY008!s & minor, G/T (%$#_{}~^\)
001NP2CP2DP2HP2SP3D=NYYYYYY008!s & minor, G/T (%$#_{}~^\)
