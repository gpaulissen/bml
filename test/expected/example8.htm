<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>No title supplied</title><style type="text/css">div.start {
    display: inline-block;
    vertical-align: top;
    width: 60px;
}
    
div.bidtable {
    width:700px;
    padding-bottom: 20px;
}    

div.bidtable ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
}

div.bidtable li ul
{
    list-style-type: none;
    margin: 0;
    padding: 0;
    margin-left: 60px;
}

div.tree {
    list-style-type: none;
    padding-bottom: 20px;
}

div.tree ul {
    list-style-type: none;
    /* vline.png */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAAKAQMAAABPHKYJAAAAA1BMVEWIiIhYZW6zAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH1ggGExMZBky19AAAAAtJREFUCNdjYMAEAAAUAAHlhrBKAAAAAElFTkSuQmCC) repeat-y;
    margin: 0;
    padding: 0;
    margin-left: 10px;
}

div.tree li.node {
    margin: 0;
    padding: 0 12px;
    line-height: 20px;
    /* node.png */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIJYVaFGwAAAARSURBVAjXY2hgQIf/GTDFGgDSkwqATqpCHAAAAABJRU5ErkJggg==) no-repeat;
}

div.tree td.node {
    vertical-align: top;
    width: 60px;
}

div.tree li.last {
    /* lastnode.png */
    background: #fff url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIIhs+gc8AAAAQSURBVAjXY2hgQIf/GbAAAKCTBYBUjWvCAAAAAElFTkSuQmCC) no-repeat;
}

div.tree li.root {
    background: #fff none no-repeat;
}

.ccolor{color:Green}
.dcolor{color:Orange}
.hcolor{color:Red}
.scolor{color:Black}
</style></head><body><div class="bidtable"><ul><li><div class="start">1NT-2<span class="ccolor">&clubs;</span></div>PUP to 2<span class="dcolor">&diams;</span><ul><li><div class="start">2<span class="dcolor">&diams;</span></div>Forced<ul><li><div class="start">Pass</div>Weak with <span class="dcolor">&diams;</span></li><li><div class="start">2<span class="hcolor">&hearts;</span></div>TRF<ul><li><div class="start">2<span class="scolor">&spades;</span></div><ul><li><div class="start">2NT</div>5+c <span class="scolor">&spades;</span>, G/T</li><li><div class="start">3<span class="ccolor">&clubs;</span><span class="dcolor">&diams;</span></div><span class="scolor">&spades;</span> &amp; minor, G/T (%$#_&empty;~^\)</li></ul></li></ul></li></ul></li></ul></li></ul></div></body></html>